### PR TITLE
fix: launch review 0.5.9

### PR DESCRIPTION
**Pull request by Xazin**
Relates: #4970 #5434

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/video/upload_video_menu.dart [hunk 1]
```diff
@@ -86,6 +94,10 @@ class _EmbedUrlState extends State<_EmbedUrl> {
       return widget.onSubmit(inputText);
     }
 
+    if (inputText.startsWith('https://youtube')) {
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/video/upload_video_menu.dart:97]: `^(https?:\/\/)?(www\.)?(youtube\.com|youtu\.be)\/`

use regex would more accurately. for example, here're some edge case
1. youtube
2. www.youtube
3. youtu.be

**Comment by LucasXu0** [comment]
The AFFocusManager is not registered in the test environment, which causes tests to fail. If it's not required in the test environment, I think we can use `maybeOf` instead.
